examples/cdip_example.ipynb

```diff
@@ -0,0 +1,631 @@
+    "\n",
+    "The `request_parse_workflow` function is your one-stop shop for pulling and parsing data. It wraps MHKiT's `request_data` function so that if a station number is passed the function will pull the buoy data from the CDIP servers and then proceed to process the data for the user. `request_parse_workflow` allows the user to slice data between a start and/or end date. Alternatively, the user may specify a single year or multiple years of interest. To reduce processing time `request_parse_workflow` will allow the user to specify parameters of interest and only return data for the specified parameters. By default, the `request_parse_workflow` will pull historic data but the user may also specify real-time data if interested. Lastly, due to the long processing time two-dimensional (2D) data is not returned by default. If __all__ 2D data is desired the boolean `all_2D_variables`  may be specified as True. It is recommended however that if 2D data is needed that the user passes that parameter in the parameters field. For a full list of 1D, 2D, and metadata variables see the CDIP reference [here](https://docs.google.com/document/d/1Uz_xIAVD2M6WeqQQ_x7ycoM3iKENO38S4Bmn6SasHtY/edit) (see BUOY VARIABLES)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processing 2D Variables:\n",
+      "\n",
+      "\n",
+      "Returned data: dict_keys(['data', 'metadata']) \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mhkit.wave.io import cdip\n",
+    "import matplotlib.pyplot as plt\n",
+    "station_number = '100'\n",
+    "start_date = '2020-04-01'\n",
+    "end_date= '2020-04-30'\n",
+    "parameters =['waveHs', 'waveTp', 'waveMeanDirection']\n",
+    "\n",
+    "data = cdip.request_parse_workflow(station_number=station_number, parameters=parameters, \n",
+    "                       start_date=start_date, end_date=end_date)\n",
+    "\n",
+    "print('\\n')\n",
+    "print(f'Returned data: {data.keys()} \\n')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Request Data from CDIP\n",
+    "   \n",
+    "In the above example `request_parse_workflow` is used to both request and parse the CDIP data using multiple MHKiT functions in a presepecfied workflow for the user. Here, we will start by running the consituient functions which make up parse data.\n",
+    "\n",
+    "To get started we will take a look at the `request_data` function and what is needed to call it. Requesting the NetCDF file is broken out to provide users flexibility by taking a modular approach to the data request. This is useful for the user to make custom workflows beyond what `request_parse_workflow` does currently and can also be used if the user prefers to work with the requested data in a tool such as xarray instead of the returned dictionary of dataframes. \n",
```

Comment:
I think this description needs updated from 'request_data' to 'request_netCDF' to match the actual function name.